framework/db/oci/ColumnSchemaBuilder.php

```diff
@@ -20,14 +20,42 @@ class ColumnSchemaBuilder extends AbstractColumnSchemaBuilder
     /**
      * @inheritdoc
      */
+    protected function buildUnsignedString()
+    {
+        return $this->isUnsigned ? ' UNSIGNED' : '';
+    }
+
+    /**
+     * @inheritdoc
+     */
+    protected function buildAfterString()
+    {
+        return $this->after !== null ? " AFTER ('{$this->after}')" : '';
```

Comment:
I wondered. What about this... except how could it be written so it doesn't depend on fixed "db" component? Thanks for the swift review btw :+1: 
```php
return ($this->after !== null) ? 
		' AFTER (' . Yii::$app->db->quoteValue($this->after) . ')' : 
		'';